neteq/src/time_stretch.rs

```diff
@@ -28,15 +28,10 @@ pub enum TimeStretchResult {
 /// Audio time-stretching processor
 pub trait TimeStretcher {
     /// Process audio samples and return the result
-    fn process(
-        &mut self,
-        input: &[f32],
-        output: &mut Vec<f32>,
-        fast_mode: bool,
-    ) -> TimeStretchResult;
+    fn process(&mut self, input: &[f32], output: &mut [f32], fast_mode: bool) -> TimeStretchResult;
```

Comment:
Better memory management (no allocations during processing)
Stack-based algorithm for finding optimal removal points is clever
More intelligent about when/where to remove samples

please keep these changes